setup.py

```diff
@@ -30,8 +30,8 @@ def get_version(rel_path):
     "pysdd": ["pysdd"],
     "gcs": ["gcspy"],
     "cpo": ["docplex"],
-    "pindakaas": ["pindakaas"],
-    "pumpkin": ["pumpkin-solver>=0.2.1"]
+    "pumpkin": ["pumpkin-solver>=0.2.1"],
+    "pindakaas": ["pindakaas==0.2.0"],
```

Comment:
I'm not actually a fan of pinning the exact version of the solver. I feel we should always be supporting the lastest version of any solver, even if the tests fail on the CI because of it.
It will at least warn us of the new version being out, and typically the fix is quite easy to do as well.

For I would change this to `pindakaas>=0.2.0`

Reply:
Not sure I agree, but I'll go with that then :)